user-guide/Standard_Apps/IDP/Advanced_configuration/Configuration.md

```diff
@@ -8,10 +8,46 @@ The *Configuration* subtab of the *Admin* tab consists of the pages detailed bel
 
 ## Network Shares
 
-On this page, you can modify the credentials and path for the configuration archive, the credentials for file transfers, and the working directories list. Note that prior to IDP 1.1.20, the file transfer credentials are mentioned under *Working Directories*.
+### DataMiner Configuration Archive
+
+The DataMiner Configuration Archive is a network share used by IDP to store configuration backups. The following settings need to be configured:
+
+- **Domain**: The domain of the account used to connect to the network share.
+- **Username**: The name of the account used to connect to the network share.
+- **Password**: The password corresponding with the username.
+- **Path**: The network path to the configuration archive.
+
+The **Connection Status** is *Connected* when the configured credentials can be used to connect to the network path. Otherwise, it is *Disconnected*.
+
+### File Transfer Credentials
+
+These are the credentials IDP will be use to transfer files between IDP and other servers in the network. This includes working directories.
+
+The credentials are required in the following cases:
+
+- When configuration backups are performed by exchanging a file location. In this case, the credentials you specify must allow full control over the file location.
+- When configuration updates are performed. In this case, the credentials must allow full control over the shared locations of the working directories.
+
+The following settings need to be configured:
+
+- **Domain**: The domain of the account used to transfer files.
+- **Username**: The name of the account used to transfer files
+- **Password**: The password corresponding with the username.
+
+> [!NOTE]
+> Prior to IDP 1.1.20, the file transfer credentials are mentioned under *Working Directories*.
+
+### Working Directories
 
 In the working directories list, you can specify a working directory on the network shares for each hosting DMA. Working directories temporarily contain files during a configuration update. A working directory must be available for each DMA in the DMS.
 
+For each working directory in the DMS, the following settings need to be configured:
```

Comment:
In the below list, the Agent Name and Directory Status are actually not something that should be configured. My mistake.

Reply:
OK to merge with my latest changes?